flask/README.rst

```diff
@@ -59,7 +61,25 @@ Start flask web server:
     $ cd YOUR_PROJECT_DIR/flask/
     $ python runserver.py
 
-Go to http://127.0.0.1:5000/ in your browser
+Go to http://127.0.0.1:5000/ in your browser.
+
+If you do like that, all data added will be stored in a in-memory SQLite database.
+If you want to keep the state, export the environment variable ENVIRONMENT:
+
+.. code-block:: bash
+
+    $ export ENVIRONMENT=development
+    $ python runserver.py
+
+Or in a single line:
+
+.. code-block:: bash
+
+    $ ENVIRONMENT=development python runserver.py
+
+For production (running in a "real" database), use the value "production" and be sure to also
```

Comment:
Nobody should ever use this demo app in a production environment :) I'd recommend removing any suggestions about the production environment and leave only development and testing.

Reply:
Done!